### ISSUE TITLE
Generate constructor with parameters only

### ISSUE THREAD

**Issue by miloush** [dotnet/roslyn#20909]
**Version Used**: 15.3.0 Preview 4

**Steps to Reproduce**:

```C#
class Program
{
    static void Main()
    {
        new Program("");
    }
}
```
and invoke Generate constructor quick fix.

**Actual Behavior**:
```C#
class Program
{
    private string v;

    public Program(string v) => this.v = v;

    static void Main(string[] args)
    {
        new Program("");
    }
}
```

**Expected Behavior**:
If the argument is not going to be a member of the class, this is quite a lot of work to get rid of the generated stuff, almost better writing it manually. I would like to be able to generate a constructor without the backing member:
```C#
class Program
{
    public Program(string v)
    {

    }

    static void Main(string[] args)
    {
        new Program("");
    }
}

**Comment by CyrusNajmabadi** [comment]
This was fixed in 15.3:

![image](https://user-images.githubusercontent.com/4564579/28336496-10f4c61e-6bb7-11e7-9d62-58179f0b4bb5.png)

**Comment by Pilchie** [comment]
Wrong bug.
